src/components/notifications/notification.jsx

```diff
@@ -0,0 +1,75 @@
+import { ReactComponent as NotificationIcon } from 'assets/notification.svg';
+import { usePointsNotifications } from 'contexts/PointsNotificationContext/PointsNotificationContext';
+import PropTypes from 'prop-types';
+import React, { useEffect, useRef, useState } from 'react';
+import Close from 'react-feather/dist/icons/x';
+import { useNavigate } from 'react-router-dom';
+import {
+  CloseButton,
+  Message,
+  NotificationMessage,
+  NotificationWrap,
+  PointsLink,
+  NotificationSvg,
+} from './styles';
+
+export function Notification({ points, id }) {
+  const { removeNotification } = usePointsNotifications();
+  const [show, setShow] = useState(true);
+  const timeoutRef = useRef();
+
+  const navigate = useNavigate();
+
+  const hide = () => {
+    clearTimeout(timeoutRef.current);
+    setShow(false);
+  };
+
+  const onAnimationEnd = () => {
+    if (!show) {
+      removeNotification(id);
+    }
+  };
+
+  useEffect(() => {
+    timeoutRef.current = setTimeout(() => {
+      setShow(false);
+    }, 6000);
+
+    return () => clearTimeout(timeoutRef.current);
+  });
+
+  return (
+    <NotificationWrap
+      tabIndex="0"
+      onAnimationEnd={onAnimationEnd}
+      className={show ? 'in' : 'out'}
+      role="alert"
+    >
+      <NotificationMessage>
+        <NotificationSvg>
+          <NotificationIcon size="16" />
+        </NotificationSvg>
+        <Message> You&#39;ve earned {points} points !</Message>
```

Comment:
```suggestion
        <Message> You have earned {points} points !</Message>
```
no need for apostrophe

Reply:
this copy from the Figma I changed `'` with `&#39;` because it's return eslint error